console/src/components/Services/Data/TableModify/ModifyActions.js

```diff
@@ -128,6 +128,57 @@ const resetPrimaryKeys = () => ({
   type: RESET_PRIMARY_KEY,
 });
 
+export const removeCheckConstraint = constraintName => (dispatch, getState) => {
+  const isOk = window.confirm('Are you absolutely sure?');
```

Comment:
use standard `getConfirmation` util fn for this with a proper message. See patterns across the code